src/jit/liveness.cpp

```diff
@@ -1815,19 +1815,17 @@ bool Compiler::fgComputeLifeLocal(VARSET_TP& life, VARSET_TP& keepAliveVars, Gen
  */
 
 #ifndef LEGACY_BACKEND
-VARSET_VALRET_TP Compiler::fgComputeLife(VARSET_VALARG_TP lifeArg,
-                                         GenTreePtr       startNode,
-                                         GenTreePtr       endNode,
-                                         VARSET_VALARG_TP volatileVars,
-                                         bool* pStmtInfoDirty DEBUGARG(bool* treeModf))
+void Compiler::fgComputeLife(VARSET_TP&       life,
+                             GenTreePtr       startNode,
+                             GenTreePtr       endNode,
+                             VARSET_VALARG_TP volatileVars,
+                             bool* pStmtInfoDirty DEBUGARG(bool* treeModf))
 {
     GenTreePtr tree;
     unsigned   lclNum;
 
-    VARSET_TP VARSET_INIT(this, life, lifeArg); // lifeArg is const ref; copy to allow modification.
-
```

Comment:
It is from "Do not copy life sets in fgComputeLife." commit. 
It was used only in code like VarSetOps::AssignNoCopy(this, life, fgComputeLife(life, tree, nextColonExit, volatileVars)).
I replaced it to just update the life and save VARSET allocation.